lib/ClangImporter/ImportName.cpp

```diff
@@ -1366,6 +1366,16 @@ static bool suppressFactoryMethodAsInit(const clang::ObjCMethodDecl *method,
           initKind == CtorInitializerKind::ConvenienceFactory);
 }
 
+static void
+addEmptyArgNamesForCxxFunc(const clang::FunctionDecl *funcDecl,
+                           SmallVectorImpl<StringRef> &argumentNames) {
```

Comment:
Our usual naming style would be to spell out Function.  It's not specific to C++ at all, so please drop CXX from the name  — you extracted this code from a function that was applying it to C functions!

Reply:
I concur. Renamed to `addEmptyArgNamesForClangFunction`.